lxd/containers.go

```diff
@@ -657,21 +691,29 @@ func containerSnapshotsGet(d *Daemon, r *http.Request) Response {
  * To do that, we'll need to weed out based on # slashes in names
  */
 func nextSnapshot(d *Daemon, name string) int {
-	base := fmt.Sprintf("%s/", name)
+	base := fmt.Sprintf("%s/snap", name)
 	length := len(base)
-	q := fmt.Sprintf("SELECT MAX(id) FROM containers WHERE type=1 AND SUBSTR(name,1,%d)=%s",
-		length, base)
-	rows, err := d.db.Query(q)
+	q := fmt.Sprintf("SELECT MAX(name) FROM containers WHERE type=1 AND SUBSTR(name,1,?)=?")
```

Comment:
What's the 1 here? I saw this elsewhere, seems kind of magical :). Can we make it a constant or something?